.travis.yml

```diff
@@ -13,6 +13,8 @@ branches:
 before_install:
 - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi $ANDROID_ABI
   -c 20M
+- echo yes | android update sdk --filter extra-android-support --no-ui --force > /dev/null
```

Comment:
The build process does not update the android support lib artefacts in the local repo and thus upgrading to a support lib version newer than 20.x fails since it can't be found in a maven repo. So this android call updates the support libs so the dependency can be resolved. It is just cosmetics. This can be removed if the gradle file is reverted back to reference support lib v20.0.0